### PR TITLE
Improve aria support in utilization bar chart

### PR DESCRIPTION
**Pull request by huerlisi**

### CODE REVIEW

File: src/charts/utilization-bar/utilization-bar-chart.html [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-      <div class="progress-bar" role="progressbar" ng-class="{'animate': animate,
+      <div class="progress-bar" aria-valuenow="{{chartData.percentageUsed}}" aria-valuemin="0" aria-valuemax="100" ng-class="{'animate': animate,
            'progress-bar-success': isOk, 'progress-bar-danger': isError, 'progress-bar-warning': isWarn}"
            ng-style="{width:chartData.percentageUsed + '%'}" tooltip="{{chartData.percentageUsed}}% Used" >
         <span ng-if="chartFooter" ng-bind-html="chartFooter"></span>
         <span ng-if="!chartFooter && (!footerLabelFormat || footerLabelFormat === 'actual')"><strong>{{chartData.used}} of {{chartData.total}} {{units}}</strong> Used</span>
         <span ng-if="!chartFooter && footerLabelFormat === 'percent'"><strong>{{chartData.percentageUsed}}%</strong> Used</span>
       </div>
-      <div class="progress-bar progress-bar-remaining" role="progressbar" aria-valuenow="5" aria-valuemin="0" aria-valuemax="100"
+      <div class="progress-bar progress-bar-remaining"
```
Comment (dtaylor113) [inline src/charts/utilization-bar/utilization-bar-chart.html]: Shouldn't this now be:

`aria-valuenow="{{100 - chartData.percentageUsed}}" aria-valuemin="0" aria-valuemax="100"`

File: test/charts/utilization-bar/utilization-bar.spec.js [hunk 2]
```diff
@@ -33,22 +33,27 @@ describe('Directive: pfUtilizationBarChart', function() {
...
+  it("should set the width of the inner bar to be 80%", function() {
     utilizationBar = angular.element(element).find('.progress-bar').css('width');
-    expect(utilizationBar).toBe("50%");
+    expect(utilizationBar).toBe("80%");
+  });
+
+  it("should set aria-valuenow values", function() {
+    used = angular.element(element).find('.progress-bar').not('.progress-bar-remaining');
+    expect(used.attr('aria-valuenow')).toBe("80");
   });
```
Comment (dtaylor113) [inline test/charts/utilization-bar/utilization-bar.spec.js]: With the above suggested changes, you should now be able to test for progress-bar-remaining values.

File: src/charts/utilization-bar/utilization-bar-chart.html (hunk 1 above)
Comment (huerlisi) [inline src/charts/utilization-bar/utilization-bar-chart.html]: Well, I've done that in 2d0f11e, but then dropped the aria-* values in 638b2d9. Reason is explained in the commit messages.

I did it explicitly in multiple commits to show how my reasoning evolved. But also to be able to drop the the last commit (638b2d9) if you don't agree with my reasoning.

**Comment by dtaylor113** [comment]
Ok, LGTM, thanks @huerlisi

**Pull request merged**